src/core/actions/index.js

```diff
@@ -35,6 +38,15 @@ export function setLang(lang) {
   };
 }
 
+export function setUserAgent(userAgent) {
+  const { browser, os } = UAParser(userAgent);
```

Comment:
It's documented in code examples in the README:

<img width="849" alt="screenshot 2017-03-10 01 25 40" src="https://cloud.githubusercontent.com/assets/90871/23778155/8a887450-0530-11e7-8bff-7f74d7d3b327.png">